typescript/.eslintrc.cjs

```diff
@@ -0,0 +1,64 @@
+/** @type {import('eslint').Linter.Config} */
+module.exports = {
+  root: true,
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 'latest',
+    sourceType: 'module',
+    warnOnUnsupportedTypeScriptVersion: true,
+  },
+  env: { node: true, browser: true },
+  reportUnusedDisableDirectives: true,
+  extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended', 'plugin:prettier/recommended'],
+  plugins: ['@typescript-eslint', 'prettier'],
+  rules: {
+    quotes: ['warn', 'single'],
+    '@typescript-eslint/no-var-requires': ['off'],
```

Comment:
Do I understand correctly that turning off this rule allows `require()`-style imports in the code? If so, maybe we shouldn't turn it off as the code already uses ESM-style `import`s?

Reply:
I just removed it now since it's better to disable the rule individually in a single file. The goal of disabling it was to have a quick check that require() fully works. Good call.

Reply:
Is this resolved? I'd like to merge if it's ready.

Reply:
I believe it is